recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ source:
     - configure.patch
 
 build:
-  number: 0
+  number: 1  # [linux]
+  number: 0  # [win or osx]
```

Comment:
I prefer not to split the build numbers, having a build missing on the platforms that your change is irrelevant for is fine IMHO. My reason is because I like to think in terms of "build X fixed problem A" and that's easier to remember than "build X on Windows, Y on Linux and Z on OSX fixed problem A".